tests/test_main.py

```diff
@@ -390,9 +394,11 @@
 
 def test_create_movie_transaction_atomic(genres_data, actors_data):
     with pytest.raises(ValueError):
```

Comment:
The `create_movie` function is expected to raise a `ValueError` when invalid genre IDs are provided. Ensure that the `create_movie` function includes validation for genre IDs and raises the appropriate exception.